.golangci.yml

```diff
@@ -37,8 +46,31 @@ issues:
   # List of regexps of issue texts to exclude, empty list by default.
   exclude-use-default: false
   exclude:
-    - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
+    # The line below blocks revive and many other linters from working.
+    # TODO(algebra8): Find solution to line below or remove if not needed.
+    # - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
```

Comment:
Can these be removed?

Reply:
I forgot about that commented out line, thank you for bringing that to my attention.

I will set the PR as WIP again and try to figure out what that line did. As far as I can tell, that line was casting a wide net of linter exclusions - which I assumed was not desired. However, since all linters are passing now it should be ok to remove.